Jenkinsfile

```diff
@@ -14,7 +14,7 @@ node {
   
   stage ("Deploy"){
     NAMESPACE = k8s.getNamespaceFromBranch(env.BRANCH_NAME)
-    if (env.BRANCH_NAME in ['rc']) {
+    if (env.BRANCH_NAME in ['rc', 'master']) {
       k8s.updateImageTag(NAMESPACE, docker2.imageTag(), 'gcr.io/cyrus-containers/kiwi', env.BRANCH_NAME)
```

Comment:
We won't be deploying to the production NAMESPACE for master so you may need to do some extra logic on this one - you can handle it in k8s-bench or here but I think it makes more sense to handle it here.

Reply:
Patched to internal-master in latest push